<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coursefind</class>
 <widget class="QWidget" name="coursefind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>YOUR COURSE                     EDIT HERE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>FIND BY COURSE ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>COURSE NAME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TEACHER:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TERM:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ADDRESS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PERIOD:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_course_IDlineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_course_namelineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_teacher_namelineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_course_timelineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_course_addresslineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_course_periodlineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RESULT:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CRIDET:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stu_course_creditlineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stu_query_courseButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stu_course_backButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>370</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_all_courseButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SHOW ALL</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>301</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>OBJECT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TEACHER</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SCORE</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="choose">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CHOOSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drop_courseButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DROP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
